lib/diagnostic.js

```diff
@@ -39,20 +51,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'cuttie',
+  species: 'collie',
+  breed: 'arabian white'
+  vocalization: 'meowwww',
+  vocalize: function() {
+    console.log('purrrrrrr')
+  }
+
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function ('name', 'species', 'breed', 'vocalization') {
+  this.name: name,
+  this.species: species,
+  this.breed: breed,
+  this.vocalization: vocalization
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.  The `vocalize` method
 // should return the pet's vocalization attribute.
 
 /* your answer here */
+const vocalize.prototype = Vocalize(meoww)
```

Comment:
You want to be adding this the `Pet prototype`, so something like this: `Pet.prototype.vocalize = function() {return this.vocalization}`